pybossa/view/projects.py

```diff
@@ -2368,6 +2369,7 @@ def task_notification(short_name):
                                pro_features=pro))
 
     remaining = form.remaining.data
+    webhook = form.webhook.data
     n_tasks = cached_projects.n_tasks(project.id)
```

Comment:
i think there should be some validation on what is being passed as webhook value at client/server side. at a bare minimum, ensuring its valid url can be added. however, there's scope to research that data passed under webhook doesn't results into any malicious activity.

Reply:
Is simply check the url starts with `http://` or `https://` enough?

Reply:
i would recommend urlparse / wtform validator / regex that validates parts of url as http://google won't turn into valid url though it begins w/ http://